### PR TITLE
Adjusted some NPC names

### PR DESCRIPTION
**Pull request by zackdreaver**
* **Addressed Issue(s)**: N/A

* **Server Mode**: Both

* **Description of Pull Request**: 
This pull request changed some NPC names and dialogues based on kRO client's quest and navi file.

1. Original kRO name for iRO version of "Cougar" is "쿠우가", which should be translated to "Kuuga"
2. Fixes https://github.com/rathena/rathena/issues/3854
3. A follow up of https://github.com/rathena/rathena/pull/3224 which misses some dialogues.

### CODE REVIEW

File: npc/re/jobs/2e/kagerou_oboro.txt [hunk 1]
```diff
@@ -844,9 +844,9 @@ job_ko,81,124,4	script	Cougar#ko	730,{
 		deletearray .@n[10],getarraysize(.@n) - .@questions;
 		freeloop(0);
 
-		set @job_ko_cougar,0;
+		set @job_ko_Kuuga,0;
```
[the remark this answers is not in the document]
Comment (zackdreaver) [reply npc/re/jobs/2e/kagerou_oboro.txt]: Does it affects something?
I don't see any error nor warning on console.

If it causes any error/ warning, I will revert it.

[the remark this answers is not in the document]
Comment (Daegaladh) [reply npc/re/jobs/2e/kagerou_oboro.txt]: Yeah, if someone's in the middle of this quest and the server gets updated, they'll get stuck or need to start the quest over.

**Comment by zackdreaver** [comment]
From quest2display
5139#낮 익은 그!!#SG_FEEL#QUE_NOIMAGE#
지식 시험장에서 쿠우가 가이를 만났다.#
쿠우가 가이가 내는 지식시험에 통과하자.

There is "쿠우가" which is translated as Kuuga.

This quest ID is used in npc\re\jobs\2ekagerou_oboro.txt

**Comment by Daegaladh** [comment]
I've checked both iRO and euRO, and the name is "Cougar-guy", but his name on kRO, as you said is Kuuga on the npc name and Kuuga Gai (쿠우가 가이) in the text.
On jRO his name is Kuuga-kai (クウガ・カイ). No idea why is kai instead of gai :/
Probably it was wrongly translated to Cougar because of Wildcat Joe (both are big cats?), and gai (가이) can also be tranlated to guy.
I hate iRO translators lol.

**Comment by zackdreaver** [comment]
Yeah maybe Kuuga Gai is the correct one, I'll check the other NPC name as well.

**Comment by zackdreaver** [comment]
From what I've found, "Wildcat Joe" is named "적 표범 조" in kRO which translated as "Red Panther Joe".
This "적" kinda ambiguous to me, can be Thief, target or enemeny.

**Comment by zackdreaver** [comment]
In kagerou_oboro.txt there is word "family", which is correct but not right to use.
in kRO it's called "일족" which can be translated to family or clan, the latter is the correct one.

**Comment by zackdreaver** [comment]
Probably lost in translation by iRO translator, I can't find this "Wildcat" reference anywhere in kRO quest file.
All of them said "적 표범 조" from questID 6015 to 6018.

**Comment by Daegaladh** [comment]
No, Wildcat is just how Kuuga calls him in the ninja quest, it appears only in the dialog. Later Joe says in the dialog that he prefers to be called "Red Leopard Joe" instead of "Wildcat Joe".
Isn't a mistranslation, is part of the quest storyline.

**Comment by zackdreaver** [comment]
I have no more things to add. Feel free to review, request or push any commit.

**Pull request merged**